mmv1/templates/terraform/examples/artifact_registry_repository_remote_custom.tf.erb

```diff
@@ -0,0 +1,31 @@
+resource "google_secret_manager_secret" "<%= ctx[:vars]['secret_resource_id'] %>" {
+  secret_id = "<%= ctx[:vars]['secret_id'] %>"
+  replication {
+    auto {}
+  }
+}
+
+resource "google_secret_manager_secret_version" "<%= ctx[:vars]['secret_resource_id'] %>_version" {
```

Comment:
Tests are failing with:

```
Error: Error creating Repository: googleapi: Error 400: An error occurred while validated upstream credentials: Artifact Registry service account "service-01234567890@gcp-sa-artifactregistry.iam.gserviceaccount.com" does not have permission to access the secret version.
```

You'll need to add this permission as part of the example, similar to https://github.com/GoogleCloudPlatform/magic-modules/blob/5923d4cb878396a04bed9beaf22a8478e8b1e6a5/mmv1/templates/terraform/examples/cloudbuild_trigger_webhook_config.tf.erb#L31

Reply:
Better example - doesn't need to use a full policy: https://github.com/GoogleCloudPlatform/magic-modules/blob/5923d4cb878396a04bed9beaf22a8478e8b1e6a5/mmv1/templates/terraform/examples/cloud_run_service_secret_environment_variables.tf.erb#L16

Reply:
Thanks for the reference! I tested without secret access permissions and it worked, so I've pushed the changes for testing on github.